.gitlab/deps_build.yml

```diff
@@ -55,3 +55,25 @@ build_clang_arm64:
   tags: ["runner:docker-arm", "platform:arm64"]
   variables:
     ARCH: arm64
+
+.build_omnibus-nikos_common:
+  stage: deps_build
+  rules:
+    !reference [.manual]
+  script:
+    - git clone https://github.com/DataDog/omnibus-nikos.git /tmp/omnibus-nikos
+    - cd /tmp/omnibus-nikos
+    - /bin/bash -l -c "bundle install"
```

Comment:
Can you explain why this needs a subshell (or why it isn't simply `bundle install`)?

Reply:
It's because of the way that rvm/bundler had to be installed on the `omnibus-nikos_x64` build image.`bundle` is not recognized as a command because it was originally installed this same way (see [here](https://github.com/DataDog/datadog-agent-buildimages/blob/cb0ed517f6a42d7c581ac0c5b584d803969244dc/omnibus-nikos_x64/Dockerfile#L49-L52)). I can't remember the specifics of why it has to be done like that, but FWIW that's how it's done on the other build images as well.